kernel_hardening_checker/engine.py

```diff
@@ -219,7 +226,9 @@ def table_print(self, mode, with_results):
     def json_dump(self, with_results):
         dump = self.opts[0].json_dump(False)
         if with_results:
-            dump.append(self.result)
+            # Add 'result_text' and 'result' keys to the dictionary
+            dump["check_result_text"] = self.result
+            dump["check_result"] = "FAIL" not in self.result
```

Comment:
I see that pylint doesn't like this line:

```
kernel_hardening_checker/engine.py:231:49: E1135: Value 'self.result' doesn't support membership test (unsupported-membership-test)
```

Reply:
I think we should better have 
```suggestion
            dump["check_result"] = self.result.startswith('OK')
```
in both cases.